### ISSUE TITLE
[Accessibility]: Code Insights Edit UI - Series checklist is not read

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41672]
### Audit type

Screen reader navigation

### Problem description

![PixelSnap 2022-09-14 at 15 14 22](https://user-images.githubusercontent.com/1855233/190253273-ccbddd35-c67c-4e7a-be27-2095f3e47614.png)

The checklist is not read. If something goes wrong with the query, the user will not know.

### Expected behavior

There should be a way to highlight and read this list.

Also, if there is a change to the query that results in an error in this list, that error should be announced.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov
